src/components/structures/UserSettings.js

```diff
@@ -444,6 +444,27 @@ module.exports = React.createClass({
         );
     },
 
+    _renderReferral: function() {
+        const teamToken = window.localStorage.getItem('mx_team_token');
+        if (!teamToken) {
+            return null;
+        }
+        if (typeof teamToken !== 'string') {
+            console.warn('Team token not a string');
+            return null;
+        }
+        const href = window.location.origin +
+            `/#/register?referrer=${this._me}&team_token=${teamToken}`;
```

Comment:
Bear in mind this won't work in the electron app as your window.href will be a file URI. Are we supporting team-based stuff in electron? I guess we can do if we ship the appropriate config.